library/general/requiredprobes/template.yaml

```diff
@@ -4,7 +4,17 @@ metadata:
   name: k8srequiredprobes
   annotations:
     metadata.gatekeeper.sh/title: "Required Probes"
-    metadata.gatekeeper.sh/version: 1.0.0
+    metadata.gatekeeper.sh/version: 1.1.0
+    metadata.gatekeeper.sh/requires-sync-data: |
```

Comment:
To clarify a bit around the UX, with the `onlyServices` parameter, it seems syncing service is optional, however does this annotation make syncing service a requirement?

Reply:
@apeabody this is a good question. How do we deal with conditionally referential templates? Should we just have a separate template, instead?

Reply:
Creating a separate template is an option, although I believe our goal should be to minimize the overall quantity of templates.

The current release Gatekeeper doesn't enforce `requires-sync-data`, and unmarshaling of the annotation was only added in [v3.13.0-beta.1](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.13.0-beta.1).  Depending on the roadmap, this might not be a blocker if the future plan was only to add a warning by default if the "required" sync wasn't configured in gatekeeper?